plugin/viewdoc_info.vim

```diff
@@ -168,26 +173,18 @@ endfunction
 function s:ParamsToNode(...)
 	if a:0 == 0
 		return '(dir)Top'
-	elseif a:0 == 1
-		if a:1 =~ '^(.\+)'
-			return a:1
-		else
-			return printf('(%s)Top', a:1)
-		endif
 	else
-		if a:1 =~ '^(.\+)'
-			return join(a:000, ' ')
-		else
-			return printf('(%s)%s', a:1, join(a:000[1:], ' '))
-		endif
+		let params =  join(map(copy(a:000), 'escape(v:val, "( )")'), ' ')
+		return system(printf('%s %s -o- | head -n 2', g:viewdoc_info_cmd, params) .
```

Comment:
And this should be:
```vim
return system(printf('%s %s -o- | head -n 2', g:viewdoc_info_cmd, shellescape(params, 1)) .
```
but you should check is your escaping of `( )` won't result in over-escaping after `shellescape()`.